### DISCUSSION TITLE
Search based employee directory doesn't show presence

### DISCUSSION THREAD

**Discussion by BrinaD1** [microsoft-search/pnp-modern-search#1941] [Q&A] [answered]
Hello, I've used the PnP search web parts to build an employee directory. I'm using the Graph Toolkit to display the person and have added in the property to show presence but it returns Offline for everyone. Looking at the API permissions in the SharePoint Admin Center I don't see an API request for Presence and I'm assuming that is why it shows as offline. Can anyone confirm this or point me to a workaround?
https://devblogs.microsoft.com/microsoft365dev/microsoft-graph-toolkit-adds-presence-templating-for-the-login-component-a-react-wrapper-preview-and-more/
```
<mgt-person 
          user-id="{{getUserEmail (slot item @root.slots.PersonQuery)}}" 
          view="threeLines"  
          person-card="hover" 
          show-presence>
</mgt-person>
```

![image](https://user-images.githubusercontent.com/5232243/158401223-937e9199-a6b0-4fd0-b34e-ee49ea5b8bb3.png)

![image](https://user-images.githubusercontent.com/5232243/158401378-34561741-6473-486e-a990-8076cde1e19e.png)

**Comment by wobba** [comment, marked as answer]
You need to add the graph scope `Presence.Read.All` to the web part and rebuild. I'll add this as an enhancement we can add by default.

**Reply by BrinaD1** [reply]
Thank you!

**Reply by VerlindenEAM** [reply]
Yes, please add this by default, thanks!

**Reply by wobba** [reply]
Added in #2252
